homeassistant/config_entries.py

```diff
@@ -128,7 +128,8 @@ def __init__(self, version: int, domain: str, title: str, data: dict,
 
     async def async_setup(
             self, hass: HomeAssistant, *,
-            integration: Optional[loader.Integration] = None, tries=0) -> None:
+            integration: Optional[loader.Integration] = None, tries: int = 0) \
```

Comment:
This is where TypeScript is nice. Setting a default value to `0` would have made the variable type `int`